### PR TITLE
Cleaning up dict and arranging words in order

### PR DESCRIPTION
**Pull request by ssarmokadam**

### CODE REVIEW

File: en-devonfw.dict [hunk 1]
```diff
@@ -1,1427 +1,1184 @@
...
+MicroProfile
+Microfocus
+Microservice
+Misconfiguration
+Mochawesome
+Modularization
+Moq
+MrChecker
+MrChecker’s
+MySql
```
Comment (suprishi) [inline en-devonfw.dict:258]: "MySQL" seems to be the correct way.

**Review by suprishi — CHANGES_REQUESTED** [review]

File: en-devonfw.dict [hunk 2]
```diff
@@ -1,1427 +1,1184 @@
...
+Vmware
+Vue
+WCAG
+WS
+WSDL
+WSL
+WTP
+WebAIM
+WebAPI
+WebApi
```
[the remark this answers is not in the document]
Comment (maybeec) [reply en-devonfw.dict:442]: What is WebAPI? Is this it a tool or is it an api for something in the web?

File: en-devonfw.dict [hunk 3]
```diff
@@ -1,1427 +1,1184 @@
...
-zA
+bundler
+bytecode
+camelCase
+cancelled
+cancelling
+capgemini
+catched
+cd
+cdk
```
[the remark this answers is not in the document]
Comment (maybeec) [reply en-devonfw.dict:549]: No I don't think so, we should only use CDK then

File: en-devonfw.dict [hunk 4]
```diff
@@ -1,1427 +1,1184 @@
...
+AX
+AdoptOpenJDK
+Aes
+Analytics
+AndHttp
+Angular's
+Ansible
+AnyConnect
+Appium
+ApplicationUser
```
[the remark this answers is not in the document]
Comment (ssarmokadam) [reply en-devonfw.dict:15]: ApplicationUser we can put in backticks but all software names or tool names we are adding to dict. So other we need to keep in dict

File: en-devonfw.dict (hunk 2 above)
[the remark this answers is not in the document]
Comment (ssarmokadam) [reply en-devonfw.dict:442]: I am removing WebAPI... it is for an api in web

File: en-devonfw.dict [hunk 5]
```diff
@@ -1,1427 +1,1184 @@
...
+accessors
+adoptopenjdk
+advices
+alertdialog
+allpackages
+alphanumeric
+analyse
+analytics
+anch
+angularcli
```
[the remark this answers is not in the document]
Comment (ssarmokadam) [reply en-devonfw.dict]: removed angularcli

File: en-devonfw.dict (hunk 3 above)
[the remark this answers is not in the document]
Comment (ssarmokadam) [reply en-devonfw.dict]: removed cdk

File: en-devonfw.dict [hunk 6]
```diff
@@ -135,7 +133,7 @@ GMailAPIConsumer
 GPL
 Generis
```
Comment (maybeec) [inline en-devonfw.dict:134]: should that be generic?

File: en-devonfw.dict [hunk 7]
```diff
@@ -255,9 +253,9 @@ Modularization
 Moq
 MrChecker
 MrChecker’s
```
Comment (maybeec) [inline en-devonfw.dict:255]: please change to simple single quote, it seems some strange quote here

File: en-devonfw.dict [hunk 8]
```diff
@@ -320,7 +318,7 @@ Postgres
 PowerShell
 PowerToys
 Prebuilt
-Prerequisities
+Prerequisites
```
Comment (maybeec) [inline en-devonfw.dict:321]: it's an ordinary word isn't it? so we can remove it.

File: en-devonfw.dict [hunk 9]
```diff
@@ -475,13 +471,11 @@ Zuul
...
 adoptopenjdk
 advices
-alertdialog
 allpackages
 alphanumeric
 analyse
 analytics
 anch
-angularcli
 anyedit
```
Comment (maybeec) [inline en-devonfw.dict:479]: AnyEdit

File: en-devonfw.dict [hunk 10]
```diff
@@ -606,7 +599,7 @@ datetime
 decrypt
 defacto
```
Comment (maybeec) [inline en-devonfw.dict:600]: de-facto

File: en-devonfw.dict [hunk 11]
```diff
@@ -870,10 +862,6 @@ mucevolve
 multibranch
 multilicensing
 mvc
-mvn
-mvnw
-mycomponent
-mysql
 mythaistar
 mythaistarrestaurant
```
Comment (maybeec) [inline en-devonfw.dict:866]: MyThaiStar

**Review by maybeec — CHANGES_REQUESTED** [review]
Please check the whole list, there is so much of actual correct words, which should be remove here. then there are some names, which are not written in correct capitals, then there are even wrongly spelled words and combined words, which should be simply splitted.

File: en-devonfw.dict [hunk 12]
```diff
@@ -255,9 +253,9 @@ Modularization
 Moq
```
[the remark this answers is not in the document]
Comment (ssarmokadam) [reply en-devonfw.dict:253]: it is library for .net (https://github.com/moq/moq4)

File: en-devonfw.dict (hunk 7 above)
Comment (ssarmokadam) [reply en-devonfw.dict]: fixed it

**Pull request merged**